src/vaadin-text-field-mixin.html

```diff
@@ -517,6 +538,12 @@
       this.value = '';
     }
 
+    _autoselect() {
```

Comment:
I would suggest using `_onFocus` here to make it clear that this function is supposed to be event listener only, and distinguish from a potential "private" method.

Reply:
Done.